patches/tModLoader/Terraria/ModLoader/Core/ModOrganizer.cs

```diff
@@ -70,7 +70,8 @@ internal static class ModOrganizer
 				.Union(Directory.GetFiles(ModLoader.ModPath, "temporaryDownload.tmod", SearchOption.TopDirectoryOnly)); // Old tML remnant
 		}
 
-		private static bool LoadSide(ModSide side) => side != (Main.dedServ ? ModSide.Client : ModSide.Server);
+		//TODO: Find a good common place for this code.
+		internal static bool LoadSide(ModSide side) => side != (Main.dedServ ? ModSide.Client : ModSide.Server);
```

Comment:
yeah, lets move `LoadSide` to a public class, how about `AutoloadAttribute`?? or like, top level loading, maybe `Mod`?